sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/AutomaticRepairsPolicy.cs

```diff
@@ -64,12 +68,20 @@ public AutomaticRepairsPolicy(bool? enabled = default(bool?), string gracePeriod
         /// suspended due to a state change on VM. The grace time starts after
         /// the state change has completed. This helps avoid premature or
         /// accidental repairs. The time duration should be specified in ISO
-        /// 8601 format. The minimum allowed grace period is 30 minutes
-        /// (PT30M), which is also the default value. The maximum allowed grace
+        /// 8601 format. The minimum allowed grace period is 10 minutes
```

Comment:
>10 [](http://example.com/codeflow?start=61&length=2)

won't this be a breaking change in the sdk?

Reply:
it is a breaking change and seeking approval at the swagger Level